copulas/multivariate/vine.py

```diff
@@ -182,19 +180,19 @@ def sample(self, num_rows=1):
                     if current_ind != -1:
                         # the node is not indepedent contional on visited node
                         copula_type = current_tree[current_ind].name
-                        copula = Bivariate(CopulaTypes(copula_type))
-                        copula.theta = current_tree[current_ind].theta
-                        derivative = copula._partial_derivative
```

Comment:
@ManuelAlvarezC Shouldn't this be `partial_derivative` instead of `_partial_derivative`?

Reply:
@csala No, it should be`_partial_derivative` because it's scalar.It's true though that `Clayton` copula doesn't have that method, so we need to fix this.

Reply:
Ok, no problem @ManuelAlvarezC Let's close this to keep it cleaner and open an separated issue to fix this.